TheMET/TheMET/ArtFilter/ArtFilter.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ArtFilter.swift
+//  TheMET
+//
+//  Created by Анна Ситникова on 10/08/2023.
+//
+
+import Foundation
+import UIKit
+
+class ArtFilter {
+    func filter(arts: [Art], searchText: String?) -> [Art] {
+        var filteredArts:[Art] = []
```

Comment:
This array is better to instantiate after the guard-check

Reply:
+